toree_install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Find the path of BigDL and Spark
+export BIGDL_PIP_HOME=`pip show BigDL | sed -n -e '/^Location/p' | sed 's/[^ ]* //'`
+export BIGDL_HOME=${BIGDL_PIP_HOME}/bigdl/share
+export PYSPARK_PIP_HOME=`pip show pyspark | sed -n -e '/^Location/p' | sed 's/[^ ]* //'`
+export SPARK_HOME=`python ${PYSPARK_PIP_HOME}/pyspark/find_spark_home.py`
+
+# Check installation of BigDL
+if [ -z "${BIGDL_HOME}" ]; then
+	echo "Please install BigDL correctly!"
```

Comment:
Change to Cannot find BigDL installation directory. Have you run 'pip install BigDL=x.x.x'

replace the x.x.x with the required bigdl version. I think it should be 0.4.0, right?